src/engine/datatype/ObjectDataType.ts

```diff
@@ -151,7 +151,7 @@ export class ObjectDataType extends ComplexDataType {
     return resultAttributes;
   }
 
-  validate: DataTypeValidateType = ({ value }) => {
+  validate: DataTypeValidateType = (value) => {
```

Comment:
I think this is a typo. 
`value` should be a prop. 😉

Reply:
I changed it because for some reason the typescript compiler was complaining about it, i'll revert it of course